tests/introspection/tests.py

```diff
@@ -107,6 +107,7 @@ def test_get_table_description_nullable(self):
         )
 
     @skipUnlessDBFeature('can_introspect_autofield')
+    @skipIf(connection.vendor == 'sqlite', 'All primary keys treated as 64-bit integers.')
```

Comment:
Rather than skip the test, I would add a database feature `introspected_big_auto_field_type = 'BigAutoField'` which could be set to `AutoField` for the SQLite backend.

Reply:
Ok, sounds sensible. Will look into that later.